lib/config.json

```diff
@@ -35,10 +35,10 @@
                 }
             }
         },
-        "hapi-pino":{
+        "hapi-pino": {
             "plugin": "require:hapi-pino",
             "options": {
-                "prettyPrint": true
+                "logEvents": "env.NODE_ENV !== 'production'"
```

Comment:
this should be a `$filter` and `logEvents` should the events to log not a bool

Reply:
The `$filter` will differentiate the configuration based on `env.NODE_ENV`; like so:
```
"options": {
                        "$filter": "env.NODE_ENV",
                        "production": {},
                        "$default": {
"logEvents": [ 'response', 'request-error' ]
}
                    }
```

Reply:
Good catch, Bryan. One small change... I think the filter should be on the `logEvents` property instead of the whole `options` object. And I think the logic is to turn off those logs in development:
```
"options": {
    "logEvents": {
        "$filter": "env.NODE_ENV",
        "$default": [ 'response', 'request-error' ],
        "development": false
    }
}
```

Reply:
What you've written does not work, though. Javascript is not evaluated that way within the json file. What I've written in my comment above should work and matches what pino expects.